02-oop-inheritance/observerPattern.js

```diff
@@ -0,0 +1,140 @@
+//Inherit function
+var inherit = function inherit(child, parent){
```

Comment:
if you store the function into a variable you might use an anonymus function to define it